.github/workflows/ci.yaml

```diff
@@ -77,19 +77,32 @@ jobs:
               agent: 3.4.4  # renovate: juju-agent-pin-minor
               allure_on_amd64: true
             architecture: arm64
-    name: Integration | ${{ matrix.juju.agent }} | ${{ matrix.architecture }}
+          - juju:
+              snap_channel: 3.6/stable
+              libjuju: 3.5.2.0
+              allure_on_amd64: false
+            architecture: amd64
+            lxd_channel: 6.1/stable
+          - juju:
+              snap_channel: 3.6/stable
```

Comment:
it is not stable, but there's a `3.6/stable` track that is the same revision as `3.6/beta`. i would rather test with the stable versions rather than beta where breaking changes are more frequent. @taurus-forever thoughts?

Reply:
it auto tracks the most stable prior track if it has not been released. see output of `snap info juju`:

```
channels:
  3/stable:      3.5.3             2024-07-26 (28060)  98MB -
  3/candidate:   ↑                                          
  3/beta:        3.6-beta2         2024-08-02 (28096) 100MB -
  3/edge:        ↑                                          
  4.0/stable:    –                                          
  4.0/candidate: –                                          
  4.0/beta:      4.0-beta4         2024-08-14 (28177)  99MB -
  4.0/edge:      4.0-beta5-e86c4bc 2024-08-21 (28257)  99MB -
  4/stable:      –                                          
  4/candidate:   –                                          
  4/beta:        4.0-beta4         2024-08-14 (28177)  99MB -
  4/edge:        ↑                                          
  3.6/stable:    –                                          
  3.6/candidate: –                                          
  3.6/beta:      3.6-beta2         2024-08-02 (28096) 100MB -
  3.6/edge:      3.6-beta3-52b8f17 2024-08-19 (28209) 100MB -
```

Reply:
actually, you may be right. `-` might indicate not released yet. will change to `3.6/beta`

Reply:
3.6 is not stable, so changed it back to `3.6/beta` in 0f57e9ae8ac25cebaedc80c5545f6416777980f2